app/cdap/api/sourcecontrol.ts

```diff
@@ -20,6 +20,11 @@ import { apiCreator } from 'services/resource-helper';
 const dataSrc = DataSourceConfigurer.getInstance();
 const basePath = '/namespaces/:namespace/repository';
 
-export const MyRepositoryApi = {
+export const MySourceControlApi = {
```

Comment:
If you're going to change the name, lets remove the 'My' - thats a weird naming convention